app/src/test/java/org/simple/clinic/phone/MaskedPhoneCallerImplementationTest.kt

```diff
@@ -34,6 +37,18 @@ class MaskedPhoneCallerImplementationTest {
     assertThat(caller.calledNumber).isEqualTo(plainNumber)
   }
 
+  @Test
+  fun `when masking is enabled then masked phone number should be called`() {
+    config = PhoneNumberMaskerConfig(maskingEnabled = true)
+
+    val caller: Caller = mock()
+    val plainNumber = "123"
+
+    maskedPhoneCaller.maskAndCall(plainNumber, caller = caller).blockingAwait()
+
+    verify(caller).call(activity, "07930684724,123%23")
```

Comment:
We should check this number into the source code.

Reply:
This is removed due to `PhoneNumberMaskerConfig` as part of a new commit.